web/src/click_handlers.js

```diff
@@ -573,6 +574,37 @@ export function initialize() {
         );
     });
 
+    function do_render_message_view_header(
```

Comment:
I somehow feel that this code is not necessary/would be better off at some other place. While I do see `do_render_buddy_list_tooltip` present in the same file, I feel that its presence over here has to do more with conditionally rendering the buddy list based on the device type.

Reply:
Hey, I updated the file in local repo and reduced the unnecessary part of the code, do you mean to say I should put this part of code/tippy in message_view_header.js??